packages/angular/cli/src/commands/update/cli.ts

```diff
@@ -387,16 +387,16 @@ export class UpdateCommandModule extends CommandModule<UpdateCommandArgs> {
         logger.info('  ' + description.join('.\n  '));
       }
 
-      const result = await this.executeSchematic(
+      const { success, files } = await this.executeSchematic(
         workflow,
         migration.collection.name,
         migration.name,
       );
-      if (!result.success) {
+      if (!success) {
         return 1;
       }
 
-      logger.info('  Migration completed.');
+      logger.info(`  Migration completed (${files.size} files modified).`);
```

Comment:
```suggestion
      logger.info(`  Migration completed (${files.size} file(s) modified).`);
```
Could optionally make it smarter and add the `s` to file if the size is greater than 1.

Also, I think we should special case zero with a more direct message that no changes were required.